servicetalk-http-router-jersey/src/testFixtures/java/io/servicetalk/http/router/jersey/CancellationTest.java

```diff
@@ -245,6 +247,13 @@ public void onSuccess(@Nullable final StreamingHttpResponse result) {
 
                 @Override
                 public void onError(final Throwable t) {
+                    if (t instanceof IllegalStateException &&
+                            t.getMessage().contains("input stream has already been closed")) {
```

Comment:
lets avoid exception message/string comparison if possible as this is subject to change, maybe localized, and flaky for verification. If we have to we could use `LocalizationMessages.ERROR_ENTITY_STREAM_CLOSED()` but I'm not sure this test should be trying to capture/fail on error messages at all as they are expected IIUC (the test races cancel with the response).